arch/arm64/boot/dts/qcom/sm7325.dtsi

```diff
@@ -15,3 +17,102 @@
 &cpu5 { compatible = "qcom,kryo670"; };
 &cpu6 { compatible = "qcom,kryo670"; };
 &cpu7 { compatible = "qcom,kryo670"; };
+
+&lpass_audiocc {
+	compatible = "qcom,sm7325-lpassaudiocc",
+		     "qcom,qcm6490-lpassaudiocc";
+	/delete-property/ power-domains;
+ };
+
+&lpass_cpu { status = "disabled"; };
+
+&lpass_dmic01_clk {
+	drive-strength = <8>;
+	bias-disable;
+};
+
+&lpass_dmic01_data {
+	bias-pull-down;
+};
+
+&lpass_dmic23_clk {
+	drive-strength = <8>;
+	bias-disable;
+};
+
+&lpass_dmic23_data {
+	bias-pull-down;
+};
```

Comment:
I'm unsure whether all this stuff, not just dmic should go into sm7325.dtsi or into the device itself.

It's per-se not really sm7325-specific, when an SM7325 device gets an Android 16 upgrade (looks like the Nothing phone 1 is already basically EOL, wtf?) the ADSP firmware from my understanding should be doing Audioreach instead of Elite, no q6afecc anymore. This opens a whole other can of worms, so not sure...